hijack/locale/ja/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,39 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: \n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2023-03-15 07:56+0900\n"
+"PO-Revision-Date: 2023-03-15 07:56+0900\n"
+"Last-Translator: Kumamoto-Hamachi <onioni9999nagare@gmail.com>\n"
+"Language-Team: \n"
+"Language: ja\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=1; plural=0;\n"
+
+#: hijack/contrib/admin/admin.py:64
+msgid "hijack user"
+msgstr "ユーザーハイジャック"
+
+#: hijack/contrib/admin/templates/hijack/contrib/admin/button.html:7
+msgid "hijack"
+msgstr "ハイジャック"
+
+#: hijack/contrib/admin/templates/hijack/contrib/admin/button.html:9
+#, python-format
+msgid "Hijack %(username)s"
+msgstr "%(username)sをハイジャック"
+
+#: hijack/templates/hijack/notification.html:8
+#, python-format
+msgid "You are currently working on behalf of <em>%(user)s</em>."
+msgstr "現在,あなたは<em>%(user)s</em>の代理として作業しています."
```

Comment:
```suggestion
msgstr "現在,あなたは<em>%(user)s</em>の代理として作業しています。"
```

My Japanese is a little rusty, but I actually implemented this check in the `msgcheck` library. So, if you feel like a ANSII dot is a more common Japanese punctuation, I can fix `msgcheck` as well ;)

Reply:
Thank you for the polite and friendly review and messages!

In Japanese, even in horizontal writing documents such as academic papers, "。" is sometimes used instead of ".". That's why I submitted the PR with this style of writing.

However, after checking the documents from the Japanese Agency for Cultural Affairs, I found that it is preferable to use the punctuation "。" even in horizontal writing.

So, as you pointed out, I will include a commit for the change !

Danke！ :cat2:

Reply:
I forgot to ask one thing.  :bow: 

Regarding the msgcheck library, would it be okay to replace the "," with "、" ? 
Or  should we keep the "," as is. (I think either way would be fine in Japanese.)